api/v1alpha1/application.go

```diff
@@ -55,6 +55,8 @@ type ApplicationSpec struct {
 	//+kubebuilder:validation:Required
 	Port int `json:"port"`
```

Comment:
Beholdes denne for backwards compatibility?

Reply:
Jepp. Det er en enkel `int`.